lms/tasks/email_digests.py

```diff
@@ -34,10 +35,10 @@ def send_instructor_email_digests(
 
     with app.request_context() as request:  # pylint:disable=no-member
         with request.tm:
-            LOG.info(
-                "send_instructor_email_digests(%r, %r, %r, override_to_email=%r)",
+            digest_service = request.find_service(DigestService)
+            digest_service.send_instructor_email_digests(
```

Comment:
This changes the celery task (and therefore also the admin page) to use the new service and therefore to actually send emails